cypress/integration/Calculator/calculator.spec.js

```diff
@@ -0,0 +1,62 @@
+/// <reference types="cypress" />
+
+
```

Comment:
/// <reference types="cypress" />

    let selectBuild = ["Prototype", "1", "2", "3", "4", "5", "6", "7", "8", "9"];

// galima visas pasikartojančias operacijas išsikelti kaip funkcijas

    function setFirstDigit(value) {
        cy.get("#number1Field").clear().type(value)
    }
    
    function setSecondDigit(value) {
        cy.get("#number2Field").clear().type(value)
    }
    
    function setOperation(value) {
        cy.get("#selectOperationDropdown").select(value)
    }
    
    function 
    
    // Visus testus sudėti į vieną for loop (tuomet pirmam buildui runnintų visi testai, po to visi antram ir t.t.)
    
    for (let i = 0; i < selectBuild.length; i++){
        describe('Testing Build '+ selectBuild[i], () => {
            beforeEach("Execute before each test",() => {
                cy.visit("https://testsheepnz.github.io/BasicCalculator")
                cy.get("#selectBuild").select(selectBuild[i])
             })
            
    //testas su iškeltom funkcijom atrodytų taip
            
        it("Checking calculating values on addition", () => {
            setFirstDigit(2)
            setSecondDigit(3)
            setOperation("Multiply")
                ...
        })
            
    //arba galima operacijų nerašyti atskirai kaip funkcijų ir surašyti į kiekvieną testą, tuomet kiekvienas atrodytų taip
            
        it("Checking calculating values on subtraction", () => {
            cy.get("#number1Field").clear()
            cy.get("#number1Field").type("3")
            cy.get("#number2Field").clear()
            cy.get("#number2Field").type("2")
            cy.get("#calculateButton").click("Subtract")
            cy.get("#numberAnswerField").should("have.value", "1")
            cy.get("#clearButton").click()
        })
            
     // ir taip kartoti per visus testavimo scenarijus       
            ...
        })
    }